subworkflows/local/grohmm.nf

```diff
@@ -0,0 +1,44 @@
+/*
+ * Run parametertuning optionally, otherwise just run transcript calling
+ */
+
+include { GROHMM_TRANSCRIPTCALLING } from '../../modules/local/grohmm/transcriptcalling/main.nf'
+include { GROHMM_PARAMETERTUNING } from '../../modules/local/grohmm/parametertuning/main.nf'
+
+/*
+ * Note meta refers to all merged files
+ */
+workflow GROHMM {
+    take:
+    bams
+    gtf
+    tuning_file
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    ch_tuning = []
```

Comment:
This seems redundant

Reply:
Ah, it's because I'm using the "sarek-style" so the parameter tuning on runs on:
```nextflow
        ext.when = { !params.skip_grohmm && !params.skip_tuning && params.assay_type == "GROseq" }
```

Reply:
So what actually happens below then? Assignment fails (no exception?) and it retains the `[]` value? Also it should be a rule to mark any processes with a comment if their execution is controlled using `ext.when`.

Reply:
Then it should be handled by an `if` statement, IMO. I was just trying it out, it does make the overall workflow "clean" but this was the exact thing I was afraid of, you have to go dig through the modules.config to see why it didn't execute.